solidity/test/random_beacon_service/TestUpgrade.js

```diff
@@ -1,222 +1,342 @@
-import {bls} from '../helpers/data';
-import {duration, increaseTimeTo} from '../helpers/increaseTime';
-import expectThrow from '../helpers/expectThrow';
-import {initContracts} from '../helpers/initContracts';
-import latestTime from "../helpers/latestTime";
-import {createSnapshot, restoreSnapshot} from "../helpers/snapshot";
-import expectThrowWithMessage from "../helpers/expectThrowWithMessage";
-const ServiceContractProxy = artifacts.require('./KeepRandomBeaconService.sol');
-const ServiceContractImplV2 = artifacts.require('./examples/KeepRandomBeaconServiceUpgradeExample.sol');
-const {expectEvent, time} = require("@openzeppelin/test-helpers");
-
-contract('KeepRandomBeaconService/Upgrade', function(accounts) {
-
-  let operatorContract, serviceContractProxy, serviceContract, serviceContractImplV2, serviceContractV2,
-    account_one = accounts[0],
-    account_two = accounts[1];
+import {createSnapshot, restoreSnapshot} from "../helpers/snapshot"
+const {BN, constants, expectEvent, expectRevert, time} = require("@openzeppelin/test-helpers")
+
+const ServiceContractProxy = artifacts.require('./KeepRandomBeaconService.sol')
+const ServiceContractImplV1 = artifacts.require('./KeepRandomBeaconServiceImplV1.sol')
+const ServiceContractImplV2 = artifacts.require('./examples/KeepRandomBeaconServiceUpgradeExample.sol')
+
+const chai = require('chai')
+chai.use(require('bn-chai')(BN))
+const expect = chai.expect
+
+contract.only('KeepRandomBeaconService/Upgrade', function(accounts) {
+
+  let proxy
+  let implementationV1
+  let implementationV2
+  
+  let initializeCallData
+
+  const admin = accounts[0]
```

Comment:
I would advise using another account than `accounts[0]`. This is the default account used by truffle in calls and we may miss something.